README.md

```diff
@@ -58,6 +58,12 @@ Install required modules:
 yarn install
 ```
 
+The extension makes requests to GitHub's graphql API. Changes to the graphql queries in lib/github/queries need to be compiled by running:
```

Comment:
We have a list of webhook events the extension needs listed above. Is this still accurate? 

> Select the following individual events: Branch or tag creation, Check runs, Pull requests, Pull request reviews, Pushes, Statuses. Enable the webhook.

Reply:
Yes it is, funnily enough we're already listing the events for actions and pushes that this change now uses but we didn't before. 🤷🏻